app/views/pages/teams.html.erb

```diff
@@ -0,0 +1,35 @@
+<div id="intro">
+    <h3>
+    SlashDeploy Teams
+    <small>
+    statuses and more
+    </small>
+    </h3>
+    
+    <p>
+    </p>
+    <% current_user.slack_accounts.each do |slack_account| %>
```

Comment:
Usually a slightly better way to do this is to set an instance variable in the controller, and then use that in the view. 

```ruby
# app/controllers/teams_controller.rb

class TeamsController < ApplicationController
  def index
    @teams = current_user.slack_teams
  end
end
```

And then you iterate over `@teams` in the template.

There's a few reasons to do this, but one of the big ones is perf, so you can "eager load" data in a single query, instead of N+1 in the view:

```ruby
# Syntax here may not be exact, but...
@teams = current_user.slack_teams.includes(slack_accounts: [:locks])
```